src/Handler/Search.hs

```diff
@@ -59,7 +59,7 @@ routeResult SearchResult{..} =
       ( PackageVersionModuleDocsR ppkgName pversion modName
       , Nothing
       )
-    DeclarationResult typeOrValue modName declTitle typeText ->
+    DeclarationResult typeOrValue modName declTitle _typeText ->
```

Comment:
I wasn't sure if _typeText was intended to be used in the future so left it as a (sort of) comment.  Can certainly change it to _.